### PR TITLE
fix white window on windows

### PR DESCRIPTION
**Pull request by ElderOrb**

### CODE REVIEW

File: examples/qml_video/main.cpp [hunk 1]
```diff
@@ -110,6 +110,7 @@ int main(int argc, char *argv[])
             pf = QVideoFrame::Format_NV12;
             break;
         default:
+            pf = QVideoFrame::Format_NV12;
```
Comment (valbok) [inline examples/qml_video/main.cpp:113]: what does frame.frame()->format show?

Comment (ElderOrb) [reply examples/qml_video/main.cpp:113]: Format_Invalid

Comment (valbok) [reply examples/qml_video/main.cpp:113]: AV_PIX_FMT_NONE ?

Comment (ElderOrb) [reply examples/qml_video/main.cpp:113]: Sorry, logged wrong info.. 

frame.frame()->format:  174

Comment (valbok) [reply examples/qml_video/main.cpp:113]: might be better to add case AV_PIX_FMT_D3D11: ?

Comment (ElderOrb) [reply examples/qml_video/main.cpp:113]: sure, sorry for late response :)

**Pull request merged**